drivers/sensor/lm75/Kconfig

```diff
@@ -50,4 +50,15 @@ config LM75_TRIGGER_THREAD_PRIO
 
 endif # LM75_TRIGGER
 
+config LM75_12BIT
+	bool "12-bit temperature resolution"
```

Comment:
given this is device-specific, it looks like this should be a devicetree property instead (i.e. the same board might have a mix of 9-bit and 12-bit sensors)

Reply:
Agreed.